src/functions/functionsDialog.component.tsx

```diff
@@ -340,7 +406,7 @@ const FunctionsDialog = (props: FunctionsDialogProps) => {
             </Grid>
           </Grid>
           <Divider orientation="vertical" flexItem />
-          <Grid item xs>
+          <Grid item sx={{ width: 650 }}>
```

Comment:
Maybe this needs to be a `maxWidth` or something as otherwise it takes up a lot of room on smaller screens.